pyproject.toml

```diff
@@ -33,8 +33,6 @@ ignore = [
     "F405", "F403",
     # lambda expression assignment
     "E731",
-    # .strip() with multi-character strings
```

Comment:
do we need to delete this? Didn't this serve the purpose in this case? (that it catches this issue)

Reply:
This is deleting it from the `ignore` list, and I think we do want to have the ruff `B005` check on the codebase right?